scripts/vaccination_alerts/utils.ts

```diff
@@ -141,3 +144,55 @@ export function getUserLocationsToAlert(
     .intersection(locationsToAlert)
     .value();
 }
+
+const alertTemplate = Handlebars.compile(
+  fs.readFileSync(
+    path.join(__dirname, 'vaccination-alert-template.html'),
+    'utf8',
+  ),
+);
+
+export function generateEmailContent(
+  emailAddress: string,
+  vaccinationInfo: RegionVaccinePhaseInfo,
+): string {
+  const fipsCode = vaccinationInfo.fips;
+  const region = regions.findByFipsCodeStrict(fipsCode);
+
+  const currentPhases = vaccinationInfo.phaseGroups
+    .filter(phaseGroup => phaseGroup.currentlyEligible)
+    .map((phaseGroup, index, allCurrentPhases) => {
+      const isMostRecentlyEligible = index === allCurrentPhases.length - 1;
+
+      return {
+        title: phaseGroup.tier
```

Comment:
random, but It might be a good idea to make the PhaseGroup a class and add things like `phaseGroup.title` or something

Reply:
Maybe on a follow up PR, but I like the idea!